build.sbt

```diff
@@ -13,3 +13,22 @@ scalacOptions in Test ++= Seq("-deprecation")
 testOptions in Test += Tests.Argument("-Djava.library.path=target/native")
 
 publishTo := Some(Resolver.file("file",  new File( "releases" )) )
+
+lazy val packageSqlite3C = taskKey[Unit]("Packages only Sqlite3C.java")
+
+packageSqlite3C := {
+  import scala.sys.process._
+  val log = streams.value.log
+  val cmd = Seq("/bin/bash", "-c", s"""cd target/scala-2.11/classes &&
```

Comment:
My only suggestion is to avoid `bash -c` and instead use `#&&` chaining built into ProcessBuilder http://www.scala-sbt.org/0.13/docs/Process.html